### PR TITLE
Require podman 4.5.0 for its --shm-size-systemd parameter

### PR DESCRIPTION
**Pull request by cbosdo**
## What does this PR change?

See the title

## Test coverage
- No tests: package dependencies

- [X] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

File: uyuni-tools.spec [hunk 1]
```diff
@@ -478,6 +479,10 @@ mkdir -p %{buildroot}%{_datarootdir}/fish/vendor_completions.d/
 %else
 %files -n %{name_ctl} -f %{name_ctl}.lang
 %endif
+
+# This is the commit message #3:
+
+Add missing changelogs
```
Comment (rjmateus) [inline uyuni-tools.spec:485]: do you want to add this in here?

File: uyuni-tools.spec [hunk 2]
```diff
@@ -100,6 +100,7 @@ Summary:        Command line tool to install and update %{productname}
 %if 0%{?suse_version}
 Requires:       (aardvark-dns if netavark)
 %endif
+Requires:       (podman >= 4.5.0 if podman)
```
Comment (deneb-alpha) [inline uyuni-tools.spec:103]: did you check if 4.5.0 is also available in the other distributions where we build uyuni-tools?

Comment (cbosdo) [reply uyuni-tools.spec:103]: no I didn't...

File: uyuni-tools.spec (hunk 1 above)
Comment (cbosdo) [reply uyuni-tools.spec:485]: oops... looks like a bad paste from other work

File: uyuni-tools.spec (hunk 2 above)
Comment (cbosdo) [reply uyuni-tools.spec:102]: I moved the requires inside the if to be on the safe side

**Review by rjmateus — APPROVED** [review]
Thank you for taking care of it

**Pull request merged**